js/component_declaration/common.ts

```diff
@@ -0,0 +1,29 @@
+import react from "react";
```

Comment:
`js/component_declaration/common.ts` looks strange. Maybe `js` is incorrect folder name now?

Reply:
Maybe `src` will be better?
Anyway, I suggest not rename it in the context of this PR.  I want to make sure that all build tasks will work correctly.

Reply:
I like `src`. Hope it will not be forgotten and left as is.